### DISCUSSION TITLE
Is SSR mode always SSR'd?

### DISCUSSION THREAD

**Discussion by ITwrx** [inertiajs/inertia#1185] [Help]
update: now that i re-read https://inertiajs.com/how-it-works (i've been away from inertia for a while and forgot...) i'm thinking my question is dumb, because what would be the point of inertia always SSRing every requested route? That would just be an unnecessarily complex way of creating a normal monolith. :)  Or am i still missing something...

original question below:

If i enable SSR, does it SSR for every request of the same page by the same client (always-on/fulltime SSR), or does it just SSR for the first request of a given route for a given client, and then CSR for subsequent requests for that route, by that client, like i'm given to understand sveltekit does (perhaps `isomorphic`,  or `hybrid` rendering, instead of simply `SSR`)?

The word `initial` in the quoted docs below makes me question this.

> Server-side rendering allows you to pre-render an initial page visit on the server, and to send the rendered HTML to the browser.

if the answer is that SSR mode is always-on/full time SSR, then i might would reword the docs to say something like:

> Server-side rendering pre-renders the html for pages on the server, and sends the rendered HTML to the browser.

Thanks

**Comment by Wulfheart** [comment]
No shame in asking this question.
I have been asking it to myself for a long time and it would be really nice to get some "official" statement on this matter.
